frontend_v2/src/app/components/challenge/challengesubmissions/challengesubmissions.component.ts

```diff
@@ -151,8 +151,8 @@ export class ChallengesubmissionsComponent implements OnInit, AfterViewInit {
    */
   apiCall: any;
 
-  columnsToDisplay = ['s_no', 'status', 'submission_result_file', 'stdout_file', 'stderr_file'];
-  columnsHeadings = ['S.No.', 'Status', 'Result File', 'Stdout File', 'Stderr File', ];
+  columnsToDisplay = ['s_no', 'status', 'execution_time', 'submitted_at', 'submission_result_file', 'stdout_file', 'stderr_file'];
+  columnsHeadings = ['S.No.', 'Status', 'Execution Time', 'Submitted At', 'Result File', 'Stdout File', 'Stderr File'];
```

Comment:
@savish28 let's move `Submitted At` to the end of the table.

Reply:
@Ram81 Done.